<launch>
    <include file = "$(find tutorial)/launch/test.launch" />
    <node pkg="main_func" type="tdk_main" name="tdk_main"  output="screen"/>
</launch>